nixos/tests/all-tests.nix

```
@@ -1041,6 +1041,7 @@ in {
   tika = runTest ./tika.nix;
   timescaledb = handleTest ./timescaledb.nix {};
   timezone = handleTest ./timezone.nix {};
+  timidity = handleTest ./timidity {};
```

```suggestion
  timidity = handleTestOn ["aarch64-linux" "x86_64-linux"] ./timidity {};
```

Maybe `handleTestOn` to restrict to the systems where the test actually works?